test/helpers.bash

```diff
@@ -80,17 +91,6 @@ function stacker_setup() {
     chown -R $SUDO_USER:$SUDO_USER .
 }
 
-function give_user_ownership() {
-   if [ "$PRIVILEGE_LEVEL" = "priv" ]; then
-      return
-   fi
-   if [ -z "$SUDO_UID" ]; then
-      echo "PRIVILEGE_LEVEL=$PRIVILEGE_LEVEL but empty SUDO_USER"
-      exit 1
-   fi
-   chown -R "$SUDO_USER:$SUDO_USER" "$@"
- }
-
```

Comment:
This doesn't appear to have been needed?  It was just the `cd` that failed when the alpine image wasn't found right?